### DISCUSSION TITLE
How are the final releases being built?

### DISCUSSION THREAD

**Discussion by user431246** [evilsocket/opensnitch#800] [Development]
Looking at the workflows, the OpenSnitch binary is being built using Github automatic building infrastructure, however, I do not see any mention of how this binary is being packaged into the respective deb and rpm releases. 

Would you consider automating the packaging of the releases?

**Comment by Crehet** [comment]
I also want to know this

**Comment by TriMoon** [comment]
https://github.com/evilsocket/opensnitch/pull/1019#issuecomment-1679337719
>:warning:
The service file as [distributed in the `deb` package](https://github.com/evilsocket/opensnitch/releases/tag/v1.6.2) still has **WRONG name** !!!
Someone needs to fix the packaging script !
